frontend/src/mixins/validation/ValidatorMixin.ts

```diff
@@ -34,52 +31,47 @@ import {
   getNonTechnicalBaugebiete,
 } from "@/utils/CalculationUtil";
 import FoerdermixModel from "@/types/model/bauraten/FoerdermixModel";
-import BedarfsmeldungFachabteilungenModel from "@/types/model/abfragevariante/BedarfsmeldungFachabteilungenModel";
+import BedarfsmeldungFachreferateModel from "@/types/model/abfragevariante/BedarfsmeldungFachreferateModel";
 
 @Component
 export default class ValidatorMixin extends Vue {
   /**
    * Prüft die komplette Abfrage vor dem Speichern
    */
 
-  public findFaultInInfrastrukturabfrageForSave(infrastrukturabfrage: InfrastrukturabfrageModel): string | null {
-    const validationMessage: string | null = this.findFaultInInfrastrukturabfrage(infrastrukturabfrage);
-    return !_.isNil(validationMessage) ? validationMessage : this.findFaultInAbfragevarianten(infrastrukturabfrage);
+  public findFaultInBauleitplanverfahrenForSave(abfrage: BauleitplanverfahrenModel): string | null {
```

Comment:
Ich denke es ist besser wenn die Methode `findFaultInBauleitplanverfahrenForSave` durch die Methode `findFaultInAbfrageForSave` aufgerufen wird.

Die Methode `findFaultInAbfrageForSave` findet dann in der Abfrageview Verwendung.

Die Methode `findFaultInAbfrageForSave` prüft dann um welche Abfrageart es sich handelt und ruft dann z.B. die Validierung `findFaultInBauleitplanverfahrenForSave` auf.

Reply:
done (während der Umsetzung kamen mir auch schon Zweifel über die Namensgebung...)